### ISSUE TITLE
MeshID generated by the invitation link is too long

### ISSUE THREAD

**Issue by Ozon2** [Ylianst/MeshCentral#1531]
Hello,
I am trying to get the invitation link to work. I am running MeshCentral using `node meshcentral --lanonly` with the default config on Linux. When i generate an invitation link and follow this link, i get :
```
To install, cut and paste the following command in a root terminal.
(wget "https://192.168.0.16/meshagents?script=1" --no-check-certificate -O ./meshinstall.sh || wget "https://192.168.0.16/meshagents?script=1" --no-proxy --no-check-certificate -O ./meshinstall.sh) && chmod 755 ./meshinstall.sh && sudo ./meshinstall.sh https://192.168.0.16 'ZnttDsBZyD9xgOdj4MQnzSO8tbAHV5hT9etWfcPLp8VwHpcu$XdmY8m1NcbJjUn$0kLFbSWlugwZCIhmRgqylCUTnCdD@NTpQi6miFdVmKFpuSkNMPDOmbq85fqb6nOt52ruMbZQCe$zYNBKlug8ezAoNtoWYw==' 
```
However this command doesn't work, it ends with the following error :
```
MeshID is not correct, must be 64 characters long.
```

**Comment by Ylianst** [comment]
This is a bug, working on a fix now.

**Comment by Ylianst** [comment]
Thanks for reporting this. I just checked in a fix and it will be in v0.5.70.
